packages/adapters/transports/discord/src/index.test.ts

```
@@ -0,0 +1,20 @@
+import { describe, expect, it } from "vitest";
+import { DiscordTransport } from "./index";
+
+export {};
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)